modules/xfem/src/base/XFEMCutElem.C

```diff
@@ -37,9 +37,13 @@ XFEMCutElem::XFEMCutElem(Elem * elem, unsigned int n_qpoints, unsigned int n_sid
   {
     _have_face_weights[i] = false;
     _physical_areafrac[i] = 1.0;
+    // changes are necessary here as well, but I have no access to
+    // Assembly
```

Comment:
This needs to get fixed.

Reply:
I guess we would need to pass in a reference to Assembly or something.

Reply:
I decided to leave as is for now and add a comment.